src/components/searchForm/searchForm.jsx

```diff
@@ -16,15 +16,21 @@ export const SearchForm = ({initialValue, onSearch}) => {
     setInputValue(e.target.value);
   };
 
+  const handleKeyPress = (e) => {
```

Comment:
nit: Form has built-in support for submit on enter, you could use a form instead:
```
const handleSearch = (e) => {
  e.preventDefault();
  ...
}

return (
<form onSubmit={handleSearch}>
<input type="text" placeholder="Search..." className="search-input">
<input type"submit />
</form>
)
```
This is not necessarily for this task, but keep that in mind for future implementations.